rust/operator-binary/src/catalog/config.rs

```diff
@@ -62,31 +57,6 @@ impl CatalogConfig {
         self.load_env_from_files.insert(env_name, file_name.into());
     }
 
-    pub fn add_configmap_property(
```

Comment:
I generally like the idea of removing unneeded code but i can foresee that we are going to need that later on.
Feel free to decide if you want to keep it or not ;)

Reply:
Yeah but even though they were pub, clippy was complaining and i did not want to mark it with dead code ....
We can just revive it once required?

Reply:
Agreed, works for me :+1: